Examples/OVHAPIWrapper-Example-watchOS/README.md

```diff
@@ -0,0 +1,11 @@
+![Swift Wrapper for OVH APIs - WatchOS Demo](https://github.com/cygy/swift-ovh/blob/master/img/sdk-demo-watchos.gif)
+
+This project shows you how to use the Swift OVH Wrapper framework into a WatchOS/iOS project
+
+## Platforms
+
+WatchOS and iOS
+
+## How to run
+
+In order to run this example, you have to copy the file `Examples/OVHAPIWrapper-Example-watchOS/Resources/Credentials-Sample.plist` to `Examples/OVHAPIWrapper-Example-watchOS/Resources/Credentials.plist` and to set the application key, application secret and consumer key relative to your own application (OVH Europe).
```

Comment:
Can you add a link to https://api.ovh.com/createToken/index.cgi?GET=/*&PUT=/*&DELETE=/*&POST=/* to help generate credentials (will request full access to the API)